editor/debugger/editor_visual_profiler.cpp

```diff
@@ -158,7 +159,7 @@ void EditorVisualProfiler::_update_plot() {
 	}
 
 	uint8_t *wr = graph_image.ptrw();
-	const Color background_color = get_theme_color("dark_color_2", "Editor");
+	const Color background_color = get_theme_color("dark_color_2", EditorStringName(Editor));
```

Comment:
It's always tricky with these to understand if SNAME is skipped on purpose here or by mistake. 😕 Feel free to leave it as is, there are probably many more of these.